data/blog/2023-year-in-review.mdx

```diff
@@ -0,0 +1,402 @@
+[Lume](https://github.com/luminous-devs/lume),
+[Flockstr](https://github.com/zmeyer44/flockstr),
+[Camelus](https://github.com/leo-lox/camelus),
+[noStrudel](https://github.com/hzrd149/nostrudel),
+[ZapThreads](https://github.com/fr4nzap/zapthreads), [NIP-44 Cryptography
+Audit](https://github.com/paulmillr/nip44), and
+[NostrAI](https://github.com/believethehype/nostrdvm), a NIP-90 Data Vending
+Machine Framework.
+
+In addition to the above grants we helped launch the [Nostr
+Design](https://nostrdesign.org/) initiative spearheaded by
+[Karnage](/about/karnage),
+[Daniele](/about/dtonon), and
+[Niel](/blog/niel-liesmons-joins-the-opensats-design-initiative).
+If you’re looking for design feedback or help, reach out to us or use the
+[#nostrdesign](https://nostr.band/?q=%23nostrdesign) hashtag.
+
+As you can see in the chart below, most of our nostr-related grants went towards
+client development, which is an area that needed a lot of support in the early
+stages of this novel protocol.
+
+![](/static/images/blog/2023-eoy/nostr-distribution.png)
+
+We are excited to see nostr grow and, slowly but surely, mature. It is
+undoubtedly still early days, so we expect a shift from clients to “other stuff”
+in the coming years. However, it is fantastic to see a censorship-resistant and
+sat-infused social media protocol taking off, and we are incredibly optimistic
+about its future. If you are working on a nostr-related open-source project,
+don’t hesitate to [apply for funding](/apply/grant) so you
+might join our growing list of grantees. If you want to support the nostr
+ecosystem, consider making a donation to The Nostr Fund:
+
+<center>
+    <DonateToNostrFundButton/>
+</center>
+
+---
+
+## Grantees
+
+After the initial wave of grants announced in July, we’ve added around 10
+grantees each month on average, totalling 90 approved grants as of this writing
+(not all of them announced yet).
+
+We have received some great feedback from grantees in 2023 already, and we can’t
+wait to see the fruits of everyone’s labor in the months and years to come. We
+hope to share select updates and progress reports from grantees at some point in
+the future. For now, our main focus remains on [project
+selection](/selection) and fund distribution.
+
+![](/static/images/blog/2023-eoy/grantees.png)
+
+Most of our grants are being paid out monthly. In sum we've sent more than seven
+billion sats to our grantees, in addition to a \$1M grant given to
+[GrapheneOS](/projects/grapheneos) at the beginning of the
+year. We will continue to support bitcoin-adjacent projects like Graphene and
+[The Tor Project](/projects/tor), as many people have to
+rely on these tools to use bitcoin in a secure and self-sovereign way.
+
+We expect to have over 100 grantees by Q2 of 2024, given past growth and pending
+grant applications. We hope to support many of our grantees for the long-term,
+as long as mission-aligned progress and future funding is a given.
```

Comment:
passive voice, maybe change to "as long as future funding permits and progress is aligned with our mission"